main.go

```diff
@@ -98,8 +102,20 @@ func main() {
 	} else if environment.IsCertManager() {
 		ctrlOptions.LeaderElection = true
 		ctrlOptions.LeaderElectionID = "5d2e944b.nmstate.io"
+	} else if environment.IsHandler() {
+		// Handler runs as a daemonset and we want that each handler pod will cache/reconcile only resources belongs the node it runs on.
```

Comment:
is this still needed now that the client asks the api server for the resources?

Reply:
What do you mean ?

Reply:
My bad I confused the controller client which has cache and the other client